website/content/docs/internals/telemetry/metrics/core-system.mdx

```diff
@@ -0,0 +1,151 @@
+---
+layout: docs
+page_title: "Telemetry reference: Core system metrics"
+description: >-
+  Technical reference for core system telemetry values.
+---
+
+# Core system telemetry
+
+Core system telemetry provides information about the operational health of your
+Vault instance.
+
+## Default metrics
+
+@include 'telemetry-metrics/vault/core/active.mdx'
+
+@include 'telemetry-metrics/vault/core/activity/fragment_size.mdx'
+
+@include 'telemetry-metrics/vault/core/activity/segment_write.mdx'
+
+@include 'telemetry-metrics/vault/core/check_token.mdx'
+
+@include 'telemetry-metrics/vault/core/fetch_acl_and_token.mdx'
+
+@include 'telemetry-metrics/vault/core/handle_login_request.mdx'
+
+@include 'telemetry-metrics/vault/core/handle_request.mdx'
+
+@include 'telemetry-metrics/vault/core/in_flight_requests.mdx'
+
+@include 'telemetry-metrics/vault/core/leadership_lost.mdx'
+
+@include 'telemetry-metrics/vault/core/leadership_setup_failed.mdx'
+
+@include 'telemetry-metrics/vault/core/license/expiration_time_epoch.mdx'
+
+@include 'telemetry-metrics/vault/core/locked_users.mdx'
+
+@include 'telemetry-metrics/vault/core/mount_table/num_entries.mdx'
+
+@include 'telemetry-metrics/vault/core/mount_table/size.mdx'
+
+@include 'telemetry-metrics/vault/core/performance_standby.mdx'
+
+@include 'telemetry-metrics/vault/core/post_unseal.mdx'
+
+@include 'telemetry-metrics/vault/core/pre_seal.mdx'
+
+@include 'telemetry-metrics/vault/core/replication/dr/primary.mdx'
+
+@include 'telemetry-metrics/vault/core/replication/dr/secondary.mdx'
+
+@include 'telemetry-metrics/vault/core/replication/performance/primary.mdx'
+
+@include 'telemetry-metrics/vault/core/replication/performance/secondary.mdx'
+
+@include 'telemetry-metrics/vault/core/replication/write_undo_logs.mdx'
+
+@include 'telemetry-metrics/vault/core/seal_internal.mdx'
+
+@include 'telemetry-metrics/vault/core/seal_with_request.mdx'
+
+@include 'telemetry-metrics/vault/core/seal.mdx'
+
+@include 'telemetry-metrics/vault/core/step_down.mdx'
+
+@include 'telemetry-metrics/vault/core/unseal.mdx'
+
+@include 'telemetry-metrics/vault/core/unsealed.mdx'
+
+## Barrier metrics
```

Comment:
core-system is a big group, I wonder if it makes sense to move Barrier and Caching metrics into storage.mdx?

Reply:
We can definitely add them to the storage index if that makes sense. Is there any harm to leaving them here as well?

Reply:
Nope.